templates/includes/footer.html

```diff
@@ -1,70 +1,68 @@
-<footer class="footer clearfix">
-    <div class="inner-wrapper">
-        <div class="footer__col">
-            <div class="four-col">
-                <h3 class="footer__title">MAAS</h3>
-                <ul class="no-bullets">
-                    <li>
-                        <a class="footer__link" href="/">Home</a>
-                    </li>
-                    <li>
-                        <a class="footer__link" href="/install">Install</a>
-                    </li>
-                    <li>
-                        <a class="footer__link" href="/how-it-works">How it works</a>
-                    </li>
-                    <li>
-                        <a class="footer__link external" href="http://docs.ubuntu.com/maas/">Docs</a>
-                    </li>
-                    <li>
-                        <a class="footer__link" href="/tour">Tour</a>
-                    </li>
-                </ul>
-            </div>
+<footer class="p-footer" role="contentinfo">
+    <div class="row">
+        <div class="col-4">
+            <h3 class="p-footer__title p-heading--six">MAAS</h3>
+            <ul class="p-list">
+                <li class="p-list__item">
+                    <a class="p-link--strong" href="/">Home</a>
+                </li>
+                <li class="p-list__item">
+                    <a class="p-link--strong" href="/install">Install</a>
+                </li>
+                <li class="p-list__item">
+                    <a class="p-link--strong" href="/how-it-works">How it works</a>
+                </li>
+                <li class="p-list__item">
+                    <a class="p-link--external p-link--strong" href="http://docs.ubuntu.com/maas/">Docs</a>
```

Comment:
None of the external link assets are loading 

![image](https://user-images.githubusercontent.com/1201618/27488850-685a3352-5830-11e7-85e0-897e418dd26e.png)